app-backend/app/src/main/scala/bucket/Routes.scala

```diff
@@ -1,7 +1,6 @@
 package com.azavea.rf.bucket
 
 import java.util.UUID
-import scala.concurrent.ExecutionContext
```

Comment:
Don't bother - we tend to just rebase everything down to one commit if it isn't too significant (which I think applies here, since it's just removing an unused import)